lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -445,6 +445,13 @@ def add_app_targets
                 end
                 # For macOS we do not code sign the appbundle because we do not code sign the frameworks either.
                 configuration.build_settings['CODE_SIGN_IDENTITY'] = '' if target.platform == :osx
+                # For iOS, we delete the target_installer empty values that get set for libraries since CocoaPods will
+                # code sign the libraries manually but for apps this is not true.
+                if target.platform == :ios
+                  configuration.build_settings.delete('CODE_SIGN_IDENTITY[sdk=appletvos*]')
```

Comment:
Is there a better way to enumerate these than hardcoding the different SDKs here?

Reply:
nope there is no current better way.